master/static/migrations/20231221163104_log-retention.tx.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE tasks
+    ADD COLUMN log_retention SMALLINT;
```

Comment:
can we store this as an interval in pg? usually makes it easier to work correctly with in other queries.